### ISSUE TITLE
Accessibile PDF Documents

### ISSUE THREAD

**Issue by slowbot** [bloom-housing/bloom#2203]
**What is this feature or what feature is this part of?**
PDF documents also need to be accessible to everyone, including screen reader users.

**Describe the solution you'd like**
Notes from SF PDF Review

**Convert the PDF to a Smart PDF**
- To make it easier for screen reader users to complete the form it should be converted to a Smart PDF so that they can complete it online.

**Color Contrast** 
- Should check color contrast levels

**Missing Document Title**
- There needs to be a PDF document title. It can easily be added using Adobe Acrobat. 
- Here are the steps to follow.
- Choose File > Properties > Description.
- Enter a title in the Title text box.
- Click Initial View, and then choose Document Title from the Show drop-down list. 
- Click OK to close the Description dialog box.

**Change Placement of Section Titles** 
- The section titles that appear to the left hand side of each section are hard to read as the user needs to tilt their head sideways to read the labels. 
- Would recommend moving the titles and placing them above each section.

**Create a Logical Reading Order**
- Create a logical reading order by tagging the the main title and section titles with the appropriate headings.
